src/stylesheets/elements/_buttons.scss

```diff
@@ -310,27 +353,27 @@
   }
   &:active,
   &.usa-button--active {
-    color: color('secondary-dark');
-    box-shadow: inset 0px 0px 0 1px color('secondary-darker');
+    color: color("secondary-dark");
+    box-shadow: inset 0px 0px 0 1px color("secondary-darker");
     .fa-layers {
       .fa-circle {
-        color: color('secondary-darker') !important;
+        color: color("secondary-darker") !important;
```

Comment:
remove important tag

Reply:
the prettyfier changed the single quotes for double but the !important was there, I am not sure of the implications of changing this code are

Reply:
This is the only way because of the order in which libraries are loaded